azure_pipelines/stages/steps/upm-package-info.yml

```diff
@@ -0,0 +1,29 @@
+stages:
+  - powershell: |
+      $packageJsonFile = "$(System.DefaultWorkingDirectory)\$(project.name)\Packages\$(package.name)\package.json"
+      Write-Host Current Project Version
+      $packageInfo = (Get-Content $packageJsonFile -Raw) | ConvertFrom-Json
+      $version = [version] $packageInfo.version;
+      Write-Host $version
+
+      Write-Host Determining latest release version...
+      $releases = "https://api.github.com/repos/XRTK/$project.name/releases"
+      [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+      $oldVersion = [version]::new([string](Invoke-WebRequest -Uri $releases -UseBasicParsing | ConvertFrom-Json)[0].tag_name)
+
+      if( $version -le $oldVersion )
+      {
+        $version = [version]::new($oldVersion.Major, $oldVersion.Minor, $oldVersion.Build + 1)
```

Comment:
look @SimonDarksideJ now it'll auto increment if the version is lower than what we've last released